h/schemas/admin_group.py

```diff
@@ -86,11 +86,13 @@ class CreateAdminGroupSchema(CSRFSchema):
         missing=None
     )
 
-    origins = colander.SchemaNode(
-        colander.String(),
+    origins = colander.SequenceSchema(
+        colander.Sequence(),
+        colander.SchemaNode(colander.String(),
+                            name='origin',
+                            validator=colander.url),
```

Comment:
We're only validating that the input is a URL here, not that it is an HTTP(S) URL. That is simply for the convenience of using the existing `colander.url` validator to start with. We're also not warning the user that paths are ignored.

Reply:
_Is_ the path ignored? Or does creating a group scoped to a URL-with-path end up meaning that the group doesn't appear on the site?

Reply:
Good question. It looks like the path is _not_ ignored at the point when checking whether the origin associated with a group matches that of a document. If a `GroupScope.origin` has a path I think the result will be that it will never match anything since the origin component of the document URL is compared against the `GroupScope.origin`.

Reply:
Yep. Because a `GroupScope.origin` is meant to be, and assumed to be, an origin, not a URL. But we've no validation of that because, until now, these things were only ever created by developers. The place to put in that origin-not-path validation is probably here in this admin form. (But I think we can do that as a separate card.)